packages/clay-form/src/__tests__/RadioGroup.tsx

```diff
@@ -5,7 +5,8 @@
  */
 
 import * as TestRenderer from 'react-test-renderer';
-import ClayRadioGroup from '..';
+import ClayRadio from '../Radio';
+import ClayRadioGroup from '../RadioGroup';
```

Comment:
I've been thinking about this, I do not know if it's very interesting to have a `ClayForm.RadioGroup` and `ClayForm.Radio`... So I'm thinking of the following, maybe we can have a generic `ClayForm.Group` that can handle the features of a `RadioGroup` and the simplest cases of a `Group`. Maybe it can be used up together with a group of checkboxes. What do you think?

Reply:
Not sure it makes sense to use `Group` for both use cases since `RadioGroup` has custom logic and props that a normal group wouldn't have. `RadioGroup` clones each child element and also allows controls the value of the selected radio.

Reply:
Yes I thought about this problem, I was just wondering if we could make it configurable, because soon we will have a `Group` for a use case of markup I predict ... so it would have more configurations and also seems a bit strange terms `Radio `and `RadioGroup`.

Reply:
"Radio group" is pretty common in terms of naming for what we are aiming for here. Even mdn docs reference it as a radio group. I think its best to keep it separate from a normal group we define via clay-css.

Reply:
Okay, makes sense. Let's get on with it. We can review this at any time.